backend/tournesol/serializers/inconsistencies.py

```diff
@@ -0,0 +1,42 @@
+from rest_framework import serializers
+from rest_framework.serializers import Serializer
+
+
+class ScoreInconsistencySerializer(Serializer):
+    """
+    Serializer for one element
+    """
+
+    inconsistency = serializers.FloatField()
+    entity_1_uid = serializers.CharField()
+    entity_2_uid = serializers.CharField()
+    criteria = serializers.CharField()
+    comparison_score = serializers.FloatField()
+    entity_1_rating = serializers.FloatField()
+    entity_2_rating = serializers.FloatField()
+
+
+class ScoreInconsistenciesSerializer(Serializer):
+    """
+    Returns the score inconsistencies and related data.
+    Used from a dictionary, not a queryset.
+    The count is exhaustive, but the results list is truncated at 100 elements.
+    """
+    mean_inconsistency = serializers.FloatField()
+    count = serializers.IntegerField()
```

Comment:
I realise that it may be more interesting to have the mean inconsistency and the count on a per_criterion basis. This way, you could see for which criteria the inconsistency is the highest. What do you think? Do you agree that I should change the API to have the mean_inconsistency and the count **per criterion**?

Reply:
Actually, that's what I am currently implementing with the cycles (the results are split per criterion).